zap/src/main/java/org/zaproxy/zap/extension/brk/BreakPanelToolbarFactory.java

```diff
@@ -640,6 +640,7 @@ public void actionPerformed(ActionEvent e) {
             setBreakAll(false);
             setBreakRequest(false);
             setBreakResponse(false);
+            Stats.incCounter("stats.ui.toolbar.button.break.continue");
```

Comment:
These actions are also triggered in the Break tab, just saying we'll not know in which toolbar they are actually being used.

Reply:
I left these as "toolbar" but changed all the others to "maintoolbar".

I tried to find a parent or ancestor method to use in a conditional but didn't have any success/